ports/stm32/boards/USBDONGLE_WB55/mpconfigboard.mk

```diff
@@ -1,9 +1,18 @@
 MCU_SERIES = wb
 CMSIS_MCU = STM32WB55xx
 AF_FILE = boards/stm32wb55_af.csv
-LD_FILES = boards/stm32wb55xg.ld boards/common_basic.ld
 STARTUP_FILE = lib/stm32lib/CMSIS/STM32WBxx/Source/Templates/gcc/startup_stm32wb55xx_cm4.o
 
+ifeq ($(USE_MBOOT),1)
+# When using Mboot all the text goes together after the bootloader
+LD_FILES = boards/stm32wb55xg.ld boards/common_bl.ld
+TEXT0_ADDR = 0x08004000
+else
+# When not using Mboot the text goes at the start of flash
+LD_FILES = boards/stm32wb55xg.ld boards/common_basic.ld
+TEXT0_ADDR = 0x08000000
+endif
+
```

Comment:
this change really belongs in a separate (its own?) commit (not with "Update rfcore.c to match ST examples.")